cmd/login.go

```diff
@@ -259,6 +259,9 @@ func getCertFromServer(cf *config.ServerConfig) (*cliclient.MasterClient, error)
 
 	var certReponse *CACertResponse
 	err = json.Unmarshal(content, &certReponse)
+	if nil != err {
```

Comment:
@superseb Nit: can we do `err != nil`? as thats the style used everywhere else.

Reply:
I took this from the code around it, but I found that is it inconsistent around in the CLI code. I adjusted the PR and created https://github.com/rancher/rancher/issues/15240